packages/flutter/lib/src/widgets/form.dart

```diff
@@ -161,6 +161,33 @@ class FormState extends State<Form> {
     }
   }
 
+  /// Returns a [Map] with a {submissionKey: value} entry for every
+  /// child [FormField] that has a non-null submissionKey.
+  Map<String, dynamic> submit() {
```

Comment:
I agree I don't really like submissionKey and submit. I was considering fieldName. I chose submit because this feature somewhat mimics the HTML form submit, but if you have any suggestions?

Reply:
I think the following name better describes this what this method do:

`submit` -> `getFormFieldValues`
`